grass7/raster/r.centroids/r.centroids.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+############################################################################
+#
+# MODULE:    r.centroids
+#
+# AUTHOR(S): Caitlin Haedrich (caitlin dot haedrich gmail com) and 
+#
+# PURPOSE:   Wrapper for r.volume. Creates vector map of centroids from a raster 
+#            of "clumps"; r.clumps creates "clumps" of data.
+#
+# COPYRIGHT: (C) 2021 by Caitlin Haedrich and the GRASS Development Team
+#
+#            This program is free software under the GNU General Public
+#            License (>=v2). Read the file COPYING that comes with GRASS
+#            for details.
+#
+#############################################################################
+
+#%module
+#% description: Creates vector map of centroids from raster of "clumps"
```

Comment:
```suggestion
#% description: Creates vector map of centroids from raster of "clumps".
```

(a dot for consistency with our description style)